### PR TITLE
vramsteg 1.1.0

### PR DESCRIPTION
**Pull request by crd**
* Update version
* Add head reference
* Add test do block

### CODE REVIEW

File: Library/Formula/vramsteg.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Vramsteg < Formula
   desc "Add progress bars to command-line applications"
   homepage "https://tasktools.org/projects/vramsteg.html"
-  url "https://taskwarrior.org/download/vramsteg-1.0.1.tar.gz"
-  sha256 "bc47e078079a845fa9c9cc5e4c9f4585402430ac6efc82ea6ff607506af8bdb9"
+  url "https://tasktools.org/download/vramsteg-1.1.0.tar.gz"
+  sha256 "9cc82eb195e4673d9ee6151373746bd22513033e96411ffc1d250920801f7037"
+  head "https://git.tasktools.org/scm/ut/vramsteg.git", :branch => "1.1.1", :shallow => false
```
Comment (bfontaine) [inline Library/Formula/vramsteg.rb]: Why can’t this be a shallow clone?

**Comment by crd** [comment]
For projects like this where the master branch reflects the most recent release, should I be explicitly providing a branch in head? I'm torn:

`head "https://git.tasktools.org/scm/ut/vramsteg.git", :branch => "1.1.1", :shallow => false`

If I don't, then someone who tries installing with --HEAD will get the same version of the tool until the formula gets stale and a new version is released. If I do, then it'll need to be updated with every release.

File: Library/Formula/vramsteg.rb (hunk 1 above)
Comment (crd) [inline Library/Formula/vramsteg.rb]: Mindless copying and pasting. I was basing this formula off another that hit the same bitbucket server and assumed it was necessary. I just tested it and it works fine. Thanks.

**Comment by bfontaine** [comment]
I think `HEAD` is meant for the cutting-edge code so for these cases use the branch.

**Comment by mikemcquaid** [comment]
> I think HEAD is meant for the cutting-edge code so for these cases use the branch.

Agreed :+1:

**Comment by crd** [comment]
Makes sense, thanks gentlemen. In that case I think this PR is ready to go.

File: Library/Formula/vramsteg.rb [hunk 2]
```diff
@@ -10,4 +11,9 @@ def install
     system "cmake", ".", *std_cmake_args
     system "make", "install"
   end
+
+  test do
+    # Check to see if vramsteg can obtain the current time as epoch
+    assert_match /^\d*$/, shell_output("#{bin}/vramsteg --now").chomp
```
Comment (mikemcquaid) [inline Library/Formula/vramsteg.rb]: The `.chomp` should be unnecessary here.

Comment (mikemcquaid) [inline Library/Formula/vramsteg.rb]: Also should be `/^\d+$/ if you're trying to match a number.

Comment (crd) [inline Library/Formula/vramsteg.rb]: Thanks Mike. Made the changes and confirmed `brew test` still passes.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**